app/controllers/schools/base_controller.rb

```diff
@@ -40,6 +40,6 @@ def set_school_cohort
       school: @school,
     )
 
-    redirect_to advisory_schools_choose_programme_path unless @school_cohort
+    redirect_to schools_choose_programme_path(cohort_id: Cohort.current.start_year) unless @school_cohort
```

Comment:
By just reading the code I'm not 100% sure of what happens if @school_cohort, and specs don't help me. Is there a #set_school_cohor view anywhere? I see the method gets overridden in other controllers without calling super, what is the app flow here?